demo.py

```diff
@@ -0,0 +1,85 @@
+import preliz as pz
+import logging
+
+_log = logging.getLogger("preliz")
+
+
+def one_iter(lower, upper, mode, mass=0.99, eps=0.005, plot=True):
+
+    """ Fits Parameters to a Beta Distribution based on the mode, confidence intervals and mass of the distribution.
+
+    Parameters
+    -----------
+    lower : float
+        Lower end-point between 0 and upper
+
+    upper : float
+        Upper end-point between lower and 1
+
+    mode : float
+        Mode of the Beta distribution between lower and upper
+
+    mass : float
+        Concentarion of the probabilty mass between lower and upper. Defaults to 0.99
+
+
+    eps : float
+        Tolerance for the mass of the distribution. Defaults to 0.005
+
+    plot : bool
+        Whether to plot the distribution. Defaults to True.
+
+
+    Returns
+    --------
+
+    dist : Preliz Beta distribution
+
+        Beta distribution with fitted parameters alpha and beta for the given mass and intervals.
+
+    """
+
+    if not 0 < mass <= 1:
+        raise ValueError("mass should be larger than 0 and smaller or equal to 1")
+
+    if upper <= lower:
+        raise ValueError("upper should be larger than lower")
+
+    alpha = 1
+    beta = 1
+    dist = pz.Beta(alpha, beta)
```

Comment:
probably we can do better than this based on user-passed values, as we do in maxent. Something like 

```python
    dist = pz.Beta()
    dist._fit_moments(  # pylint:disable=protected-access
        mean=(upper+lower)/2, 
        sigma=((upper - lower) / 6) / mass
    )
    tau_a = (dist.alpha - 1) / mode
    tau_b = (dist.beta - 1) / (1-mode)
    tau = (tau_a + tau_b) / 2
```
this should be a better guessing than tau=0. Maybe we can do something even better not sure yet.

then inside of the while loop, we should allow tau to increase or decrease (in case our initial guessing is higher than it should be). Something like

```python
        if prob < mass:
            tau += 0.1
        else:
            tau -= 0.1
```
On average this guessing should make the algorithm faster